### PR TITLE
display alan version

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: cli/src/deploy/mod.rs [hunk 1]
```diff
@@ -131,25 +136,25 @@ pub async fn new(agz_file: &str, cloud_alias: &str) {
   let sp = SpinnerBuilder::new(format!("Creating new app in {}", cloud_alias)).start();
   let resp = post_v1("new", body).await;
   let res = match resp {
-    Ok(app_id) => format!("Created app with id {} in {} succesfully!", app_id, cloud_alias),
+    Ok(cluster_id) => format!("Created app with id {} in {} succesfully!", cluster_id, cloud_alias),
```
Comment (dfellis) [inline cli/src/deploy/mod.rs:139]: Not in this PR, but nit: `succesfully` -> `successfully`

File: cli/src/deploy/mod.rs [hunk 2]
```diff
@@ -105,16 +110,16 @@ fn get_app_str(agz_file: &str) -> String {
...
-    "appId": app_id,
+    "clusterId": cluster_id,
   });
-  let sp = SpinnerBuilder::new(format!("Terminating app {} if it exists", app_id)).start();
+  let sp = SpinnerBuilder::new(format!("Terminating app {} if it exists", cluster_id)).start();
   let resp = post_v1("terminate", body).await;
   let res = match resp {
-    Ok(_) => format!("Terminated app {} succesfully!", app_id),
-    Err(err) => format!("Failed to terminate app {}. Error: {}", app_id, err),
+    Ok(_) => format!("Terminated app {} succesfully!", cluster_id),
```
Comment (dfellis) [inline cli/src/deploy/mod.rs:121]: Same here

File: cli/src/deploy/mod.rs [hunk 3]
```diff
@@ -131,25 +136,25 @@ pub async fn new(agz_file: &str, cloud_alias: &str) {
...
+    "clusterId": cluster_id,
     "agzB64": app_str,
   });
-  let sp = SpinnerBuilder::new(format!("Upgrading app {} with {}", app_id, agz_file)).start();
+  let sp = SpinnerBuilder::new(format!("Upgrading app {} with {}", cluster_id, agz_file)).start();
   let resp = post_v1("upgrade", body).await;
   let res = match resp {
-    Ok(_) => format!("Upgraded app {} succesfully!", app_id),
-    Err(err) => format!("Failed to upgrade app {} with {}. Error: {}", app_id, agz_file, err),
+    Ok(_) => format!("Upgraded app {} succesfully!", cluster_id),
```
Comment (dfellis) [inline cli/src/deploy/mod.rs:156]: And here

File: cli/src/deploy/mod.rs (hunk 1 above)
Comment (depombo) [reply cli/src/deploy/mod.rs:139]: oh thanks, fixing!

**Review by dfellis — APPROVED** [review]

**Pull request merged**